iOS/RemoteCollectionController.swift

```diff
@@ -9,18 +9,36 @@ class RemoteCollectionController: UICollectionViewController, DateScrubberDelega
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        for family: String in UIFont.familyNames()
+        {
```

Comment:
Could you move the `{` to be inline with the `for` declaration?

`for family: String in UIFont.familyNames() {`